frontend/zulu-app/App.js

```diff
@@ -0,0 +1,36 @@
+import { NavigationContainer } from '@react-navigation/native';
```

Comment:
I think its more  common to have filenames at lower case (also in react) - worth checking though.